config.template

```diff
@@ -1,88 +1,109 @@
 [case]
-# ACME case configuration
-casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00
-native_res = ne30
-short_term_archive = 0
-ref_casename_v0 = B1850C5_ne30_v0.4
+
+# casename is a name that identifies the simulation being analyzed.
+casename = casename
+# ref_casename is the name of a reference run to compare agains (or None
+# to turn off comparison with a reference, e.g. if no reference case is
+# available)
+ref_casename = None
```

Comment:
@milenaveneziani, thanks for the clarification here.  I think we should discuss this in our call tomorrow.

My feeling is that there should be as little as possible in both the config file and in the analysis itself that is ACME specific.  Instead, it might make sense to support analysis for a reference run that has the same formatting at the main run (e.g. a run directory and corresponding namelist and streams files) and also for another preprocessed format, of which v0 runs would be one example.  Let's discuss what the appropriate option names would be and which section(s) they would go into.